UnityProject/Assets/Scripts/UI/Core/Net/Elements/NetListButBetter.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection.Emit;
+using Newtonsoft.Json;
+using UnityEngine;
+
+public class NetListButBetter<T> : NetUIStringElement
```

Comment:
If its better why not replace old one?

Reply:
the old netlist is weird and is designed to work with dynamically loading net elements, this doesn't support dynamically loading net elements but just gives you usual Data Elements